recipe/meta.yaml

```diff
@@ -252,6 +260,7 @@ outputs:
       run:
         - {{ pin_subpackage('_r-xgboost-mutex', exact=True) }}
         - {{ pin_subpackage('libxgboost', max_pin='x.x.x') }}
+        - libgomp                          # [win]
```

Comment:
This is already added via `libgomp`'s `run_exports/strong` and it is in the top-level. So let's drop this

```suggestion
```